src/kernels/backproject.cl

```diff
@@ -21,8 +21,6 @@ constant sampler_t volumeSampler = CLK_NORMALIZED_COORDS_FALSE |
                                    CLK_ADDRESS_CLAMP |
                                    CLK_FILTER_LINEAR;
 
-#define PI 3.1415926535897932384626433832795028841971693993751058209749445923078164062f
```

Comment:
I don't think this was defined because of performance reasons but lack of reading the reference ;-)